charts/datagovuk/templates/find/deployment.yaml

```diff
@@ -16,6 +16,18 @@ spec:
         runAsUser: 900
         runAsGroup: 900
         fsGroup: 900
+      initContainers:
+        - name: config-set
+          securityContext:
+            runAsUser: 0
```

Comment:
Optional: don't worry about this for this PR, but you won't be able to run this as root in prod (or if you can, it's only because we've missed your namespace out of our pod security admission spec 😅)

Reply:
ok, I think I just left it as root in order to remove the pid, will look at fixing the in the future